tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: add-fbc-contribution
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to create internalrequests to add contributions to index images
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+      default: "snapshot_spec.json"
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+      default: "data.json"
+    - name: binaryImage
+      type: string
+      description: binaryImage value updated by update-ocp-tag task
+    - name: fromIndex
+      type: string
+      description: fromIndex value updated by update-ocp-tag task
+    - name: requestTimeout
+      type: string
+      default: "180"
+      description: InternalRequest timeout
+  results:
+    - name: fbcOptIn
+      description: Result to set weather the contribution is FBC opt in
+    - name: requestResultsFile
+      description: Internal Request results file
+    - name: requestMessage
+      description: Internal Request message
+    - name: requestReason
+      description: Internal Request reason
+  workspaces:
+    - name: data
+      description: workspace to read and save files
+  steps:
+    - name: add-contribution
+      image: quay.io/hacbs-release/release-utils:a9603c0ae1982be5e1142d44765b1475adc7ad40
+      script: |
+        #!/usr/bin/env sh
+        #
+        set -e
+
+        SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No valid data file was provided."
+            exit 1
+        fi
+
+        # Set the value for requestResultsFile
+        echo "$(workspaces.data.path)/ir-$(context.taskRun.uid)-result.txt" > $(results.requestResultsFile.path)
+
+        # IIB is the default internal pipeline for fbc requests
+        request=$(jq -r '.fbc.request // "iib"' ${DATA_FILE})
+
+        default_build_timeout_seconds="300"
+        iib_overwrite_from_index_credential=$(jq -r \
+          '.fbc.iibOverwriteFromIndexCredential // "iib-overwrite-fromimage-credentials"' ${DATA_FILE})
+        iib_service_config_secret=$(jq -r '.fbc.iibServiceConfigSecret // "iib-services-config"' ${DATA_FILE})
+        iib_service_account_secret=$(jq -r '.fbc.iibServiceAccountSecret // "iib-service-account"' ${DATA_FILE})
+        build_tags=$(jq -r '.fbc.buildTags // "[]"' ${DATA_FILE})
```

Comment:
This is a bit weird. I guess technically it works. But I assume `.fbc.buildTags` would be a list, if defined, right? So we either return a json list, or we return a `"[]"` string, but thanks to the `-r`, there will be no `""` and so it will actually look like an empty list json.

So I would suggest to drop the `-r` and the `""`: `build_tags=$(jq '.fbc.buildTags // []' ${DATA_FILE})`

Same applies to the next line.

Reply:
changed.